### ISSUE TITLE
Enable publication state to be changed via CSV imports

### ISSUE THREAD

**Issue by karenmajewicz** [geobtaa/geoblacklight_admin#23]
Publication State is now importable and exportable. However, I don't think the import is working. Itested importing a set to change records from 'unpublished' to 'published' but the state did not change.

**Comment by ewlarson** [comment]
Ahhh... Yes, you cannot change state that way, as state is held in the document_transitions table. You can only import the current state.

**Comment by ewlarson** [comment]
We might be able to fire a callback to ensure state is sync'd after import tho... I'll think it over.

**Comment by karenmajewicz** [comment]
OK - otherwise, if we cannot change the state this way, we should remove it from the import table.

**Comment by ewlarson** [comment]
I think the answer here is to just simplify publication state and NOT track document transitions between draft => published => unpublished. State tracking is nice for long running jobs like URI analysis, but it's far less helpful if you want to import publication state.

I think we refactor publication state here to just be a select option on the document table itself.

**Comment by karenmajewicz** [comment]
I tested this on dev and it mostly works as expected.  I do want to document this, however:

If the Publication State field is omitted from the import CSV, the item will revert to a "draft" state. Was this intentional  behavior?

**Comment by ewlarson** [comment]
Yes, if no publication state is present, the logic defaults to "draft". Do you need something else?

**Comment by karenmajewicz** [comment]
I think it can work - I just have to make sure to update our import templates, which currently lack the field.
